### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by BjoernAtBosch**
Fixes:
* Remove duplicate upgrade-cli.sh from common cpp setup scripts
* Build-system: Do not copy python scripts and conan and cmake folders into workspace folder
* Remove installing (i.e. potentially building) C++ dependencies on velocitas init
* Fix some misleading comments

### CODE REVIEW

File: build-system/cpp-cmake-conan/install/build.sh
Comment (MP91) [inline build-system/cpp-cmake-conan/install/build.sh:1]: why did you move these files into a new folder?

Comment (BjoernAtBosch) [reply build-system/cpp-cmake-conan/install/build.sh:1]: Because these files are the only ones, which are installed into the root folder.
I tried explicitly copying both in the manifest.json but that didn't work

Comment (doosuu) [inline build-system/cpp-cmake-conan/install/build.sh:1]: Can't we remove these files altogether by now?

File: build-system/cpp-cmake-conan/install/install_dependencies.sh
Comment (doosuu) [inline build-system/cpp-cmake-conan/install/install_dependencies.sh:1]: Can't we remove these files altogether by now?

File: build-system/cpp-cmake-conan/install/build.sh
Comment (BjoernAtBosch) [reply build-system/cpp-cmake-conan/install/build.sh:1]: We wanted to keep them for convenience. Typing `./build.sh` is faster than `velocitas exec build-system build` - especially if using auto-completion

File: build-system/cpp-cmake-conan/install/install_dependencies.sh
Comment (BjoernAtBosch) [reply build-system/cpp-cmake-conan/install/install_dependencies.sh:1]: See above

**Review by doosuu — APPROVED** [review]
lgtm

**Comment by MP91** [comment]
Are we now only syncing the 2 scripts? don't we also need the conan-profiles?

**Comment by BjoernAtBosch** [comment]
> Are we now only syncing the 2 scripts? don't we also need the conan-profiles?

Yes, only the shell scripts are synced now. For building the conan-profiles of the package are used ...
